extensions/aproc/proc/enrich/drivers/impl/safe.py

```diff
@@ -96,9 +93,7 @@ def __download_TCI(self, href: str):
             # Clean-up
             os.remove(tmp_file)
```

Comment:
If you could update the existing code with the comment "!DELETE!" please and add the comment
"remove the file that has been created within this function".